crates/rspack_napi/src/threadsafe_function/mod.rs

```diff
@@ -25,6 +28,16 @@ pub struct ThreadsafeFunction<T: 'static, R> {
   _data: PhantomData<R>,
 }
 
+impl<T, R> Debug for ThreadsafeFunction<T, R> {
+  fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+    f.debug_struct("ThreadsafeFunction")
```

Comment:
This is not intentional, right? Seems not necessary.

Reply:
This is to make it easier for structs/enums that contain  `ThreadsafeFunction`  to implement `Debug`, from `ThreadSafeFunctionWithRef` all the way up to `Filename`.

Reply:
Pointers should not be exposed. Printing `ThreadsafeFunction(...)` seems enough if you would like to implement `Debug`.

Reply:
Resolved in d9ff5eea6c77da93af96d2fdd22af158725b788b